data-contract/schema.json

```diff
@@ -0,0 +1,118 @@
+{
+  "Project": {
+    "type": "object",
+    "properties": {
+      "name": {
+        "position": 0,
+        "type": "string",
+        "description": "Project name",
+        "maxLength": 63
+      },
+      "description": {
+        "position": 1,
+        "type": "string",
+        "description": "Project description",
+        "maxLength": 1000
+      },
+      "url": {
+        "position": 2,
+        "type": "string",
+        "description": "Project URL",
+        "maxLength": 255
+      }
+    },
+    "required": [
+      "name",
+      "description",
+      "url",
+      "$createdAt",
+      "$updatedAt"
+    ],
+    "additionalProperties": false
+  },
+  "Tasks": {
+    "type": "object",
+    "properties": {
+      "title": {
+        "position": 0,
+        "type": "string",
+        "description": "Task title",
+        "maxLength": 63
+      },
+      "description": {
+        "position": 1,
+        "type": "string",
+        "description": "Task description",
+        "maxLength": 1000
+      },
+      "url": {
+        "position": 2,
+        "type": "string",
+        "description": "Task URL",
+        "maxLength": 255
+      },
+      "assignee": {
+        "position": 3,
+        "type": "array",
+        "description": "Task assignee executor",
+        "byteArray": true,
+        "minItems": 32,
+        "maxItems": 32
+      },
+      "projectId": {
+        "position": 4,
+        "type": "array",
+        "byteArray": true,
+        "minItems": 32,
+        "maxItems": 32
+      },
+      "status": {
+        "position": 5,
+        "type": "string",
+        "description": "Task status",
+        "enum": [
+          "pending",
+          "in_progress",
+          "completed",
+          "cancelled",
+          "paid"
+        ]
+      }
+    },
+    "required": [
+      "title",
+      "projectId",
+      "$createdAt",
+      "$updatedAt"
+    ],
+    "additionalProperties": false
+  },
+  "Claim": {
+    "type": "object",
+    "properties": {
+      "taskId": {
+        "position": 0,
+        "type": "array",
+        "byteArray": true,
+        "minItems": 32,
+        "maxItems": 32
+      },
+      "amountCredits": {
+        "position": 1,
+        "type": "number"
+      },
+      "amountUSD": {
+        "position": 2,
+        "type": "number"
+      }
```

Comment:
Missing something like `deliverable` that will contain a link to work being submit

Reply:
added